spec/presenters/api/v0/base_presenter_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.